src/app/features/conditions/edit-conditions/conditions.component.ts

```diff
@@ -51,6 +52,16 @@ export class ConditionsComponent extends SingleRowEditorComponent<Conditions> {
     return this.conditionType === CONDITION_TYPES.CONDITION_QUESTSTATE;
   }
 
+  get showReactionTo(): boolean {
+    switch (this.conditionType) {
+      case CONDITION_TYPES.CONDITION_REPUTATION_RANK:
+      case CONDITION_TYPES.CONDITION_REACTION_TO:
+        return true;
+      default:
+        return false;
+    }
+  }
```

Comment:
@Kitzunu I've updated this part to enable it also for `CONDITION_REPUTATION_RANK` since it's the same flag

Reply:
Yea I thought of doing that too :D